### ISSUE TITLE
[postgres only] exception on each pages including the block

### ISSUE THREAD

**Issue by mouneyrac** [fernandooliveira/moodle-block_marking_manager#7]
First I just want to say: great job! It is always nice to see a new require_grading plugin and this one even implement fast grading! 

Default exception handler: Error reading from database Debug: ERROR:  invalid input syntax for type numeric: ""\nLINE 5: ...             AND (qa.sumgrades IS NULL OR qa.sumgrades = '')\n                                                                    ^\nSELECT COUNT(DISTINCT qa.userid)\n                  FROM mdl_quiz_attempts qa\n                 WHERE qa.quiz = $1\n                   AND qa.state = 'finished'\n                   AND (qa.sumgrades IS NULL OR qa.sumgrades = '')\n[array (\n  0 => '112033',\n)]\nError code: dmlreadexception\n* line 443 of /lib/dml/moodle_database.php: dml_read_exception thrown\n* line 244 of /lib/dml/pgsql_native_moodle_database.php: call to moodle_database->query_end()\n* line 764 of /lib/dml/pgsql_native_moodle_database.php: call to pgsql_native_moodle_database->query_end()\n* line 1476 of /lib/dml/moodle_database.php: call to pgsql_native_moodle_database->get_records_sql()\n* line 1549 of /lib/dml/moodle_database.php: call to moodle_database->get_record_sql()\n* line 1759 of /lib/dml/moodle_database.php: call to moodle_database->get_field_sql()\n* line 203 of /blocks/fn_marking/lib.php: call to moodle_database->count_records_sql()\n* line 717 of /blocks/fn_marking/lib.php: call to quiz_count_ungraded()\n* line 3091 of /blocks/fn_marking/lib.php: call to count_unmarked_activities()\n* line 308 of /blocks/fn_marking/block_fn_marking.php: call to fn_build_ungraded_tree()\n* line 120 of /blocks/fn_marking/block_fn_marking.php: call to block_fn_marking->get_frontpage_content()\n* line 785 of /blocks/moodleblock.class.php: call to block_fn_marking->get_content()\n* line 236 of /blocks/moodleblock.class.php: call to block_list->formatted_contents()\n* line 1002 of /lib/blocklib.php: call to block_base->get_content_for_output()\n* line 1054 of /lib/blocklib.php: call to block_manager->create_block_contents()\n* line 369 of /lib/outputrenderers.php: call to block_manager->ensure_content_created()\n* line 26 of /theme/gourmet/layout/frontpage.php: call to core_renderer->standard_head_html()\n* line 885 of /lib/outputrenderers.php: call to include()\n* line 815 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()\n* line 101 of /index.php: call to core_renderer->header()\n, referer: https://test.vlacs.org/admin/search.php?query=debug

basically it's the quiz sql: qa.sumgrades = ''
It is not possible to check a numeric field against '' in postgres

**Comment by fernandooliveira** [comment]
Fixed.
